setup.py

```diff
@@ -29,7 +30,10 @@
                'Topic :: Internet :: WWW/HTTP',
                'Topic :: Security :: Cryptography',
                'Topic :: Software Development :: Libraries :: Python Modules']
-INSTALL_REQUIRES = ['Django>=1.11', 'fido2', 'sqlparse']  # sqlparse is required for Django < 2.2
+# FIDO 2 version specifier
+FIDO2_SPECIFIER = 'fido2=={}'.format(os.environ['FIDO2_VERSION']) if os.environ.get('FIDO2_VERSION') else 'fido2'
+print("INSTALLING {}".format(FIDO2_SPECIFIER))
+INSTALL_REQUIRES = ['Django>=1.11', FIDO2_SPECIFIER, 'sqlparse']  # sqlparse is required for Django < 2.2
```

Comment:
We agreed on forfeit this hacky solution in favour of separate tox environments.

Reply:
Fixed.